packages/orm/src/logging/ReactionLogger.ts

```diff
@@ -1,8 +1,8 @@
 import ansis from "ansis";
-import { ReactiveField } from "../config";
+import { ReactiveActor } from "../config";
```

Comment:
The term "actor" is often used to mean "erlang actors, akka actors, etc" -- maybe `Reactable`? ... `Reactive...` thinking 🤔

Reply:
Eh, yeah the terms of pretty unique though, so doing case-sensitive 

* `ReactiveActor` -> `Reactable`
* `reactiveActors` -> `reactables`

Etc with global find/replace (I used the `sd` command line utility last time, as an easier sed) seems easy enough.

It's fair to sit on it, i.e. wait & see if `Reactable` is still the best term after some thinking, but would like to see the rename happen at the last minute/before merging.